src/components/schemas/statements.yaml

```diff
@@ -0,0 +1,95 @@
+preview_in:
+  type: object
+  properties:
+    id:
+      allOf:
+        - $ref: './fields.yaml#/id_string'
+      example: stmtadfdsdf
+    statement_mapping:
+      type: object
+      properties:
+        description_column:
+          type: string
+          description: |
+            Specifies the header name of the description column in the statement.
```

Comment:
Pls use the same description as below (or) update all the below descriptions.

Ex: `Specifies the mapping of description column on the statement`